### PR TITLE
Fix menu and footer broken links in all languages (checked all .po files in /po/common/)

### PR DESCRIPTION
**Pull request by lan33**
**Description:**

This PR tries to fix broken links in menu and footer in all languages
Links should not be translated.
Alls `.po` files in `/po/common/` are checked, translated links are removed.

**Related issues and discussion:**

This fix may be related to the bug #1068

### CODE REVIEW

File: po/common/mr.po [hunk 1]
```diff
@@ -23,7 +23,7 @@ msgstr "mr"
 #. put here the language name ("French", "Japanese", etc)
 msgctxt ":langname"
 msgid ":langname"
-msgstr "भाषानाव"
+msgstr "Marathi"
```
Comment (hangy) [inline po/common/mr.po]: I am not quite sure which language `:langname` should be in. In production, we have some cases where the name is in the native language of the file, and others where it is in English. More importantly, I do not see `:langname` being used outside of loading the translations. Am I missing something, @stephanegigandet @maddingue?

Comment (teolemon) [reply po/common/mr.po]: No use for Crowdin. @maddingue  ?

Comment (stephanegigandet) [reply po/common/mr.po]: We could remove it then. It was generated when we transformed I18N.pm to .po files but it's not used.

Comment (stephanegigandet) [reply po/common/mr.po]: Filed https://github.com/openfoodfacts/openfoodfacts-server/issues/1553

**Pull request merged**